packages/x-date-pickers/src/internals/hooks/useStaticPicker/useStaticPicker.tsx

```diff
@@ -58,7 +58,15 @@ export const useStaticPicker = <
           {...slotProps?.layout}
           slots={slots}
           slotProps={slotProps}
-          sx={sx}
+          sx={[
```

Comment:
Am I missing something or does just removing the `sx` prop picking on L34 fix the problem in a cleaner way? 🤔 
By the way, wouldn't the static range picker have the same issue as well? 🤔